src/Package/Impl/ExportDialog/ExportImageDialog.xaml.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Windows;
+using System.Windows.Controls;
+using System.ComponentModel;
+using Microsoft.Common.Core;
+using Microsoft.VisualStudio.R.Package.Wpf;
+
+namespace Microsoft.VisualStudio.R.Package.ExportDialog {
+    public partial class ExportImageDialog : PlatformDialogWindow, INotifyPropertyChanged {
+
+        private int _userWidth;
+        private int _userHeight;
+        private bool _viewPlot;
+        private const int MIN = 100;
+        private const int MAX = 100000;
+        private int _prevValidWidth;
+        private int _prevValidHeight;
+        private bool _maintainAspectRatio;
+        private int _originalWidth;
+        private int _originalHeight;
+
+        public ExportImageDialog(ExportArguments imageArguments) {
+            InitializeComponent();
+            this.HasMaximizeButton = false;
+            this.HasMinimizeButton = false;
+            UserWidth = imageArguments.PixelWidth;
+            UserHeight = imageArguments.PixelHeight;
+            this.DataContext = this;
+        }
+
+        public int UserWidth {
+            get { return _userWidth; }
+            set {
+                _userWidth = value;
+                OnPropertyChanged("UserWidth");
+            }
+        }
+
+        public int UserHeight {
+            get { return _userHeight; }
+            set {
+                _userHeight = value;
+                OnPropertyChanged("UserHeight");
+            }
+        }
+
+        public bool ViewPlotAfterSaving {
+            get { return _viewPlot; }
+            set {
+                _viewPlot = value;
+                OnPropertyChanged("ViewPlotAfterSaving");
+            }
+        }
+
+        public bool MaintainAspectRatio {
+            get { return _maintainAspectRatio; }
+            set {
+                _maintainAspectRatio = value;
+                if (_maintainAspectRatio) {
+                    _originalWidth = UserWidth;
```

Comment:
Use properties instead of fields, otherwise there will be no `PropertyChanged` event raised.